e2e/app_test.go

```diff
@@ -129,6 +130,42 @@ func TestAppDashStat(t *testing.T) {
 	}
 }
 
+func TestAppProxy(t *testing.T) {
+
+	testcases := []struct {
+		name string
+		flag string
+	}{
+		{
+			name: "without no-verify-ssl flag",
+			flag: "",
+		},
+		{
+			name: "with no-verify-ssl flag",
+			flag: "--no-verify-ssl",
+		},
+	}
+	for _, tc := range testcases {
+		tc := tc
+		t.Run(tc.name, func(t *testing.T) {
+			httpProxy, cleanupProxy := proxyFake()
+			defer cleanupProxy()
```

Comment:
Please add newlines when necessary for readability. This function code is not easy on the eye :)